service/bulk_create.go

```diff
@@ -171,7 +171,7 @@ func parseSources(reqSources []m.BulkCreateSource, tenant *m.Tenant, userResourc
 		// validate the source request
 		err = ValidateSourceCreationRequest(dao.GetSourceDao(&dao.RequestParams{TenantID: &tenant.Id}), &source.SourceCreateRequest)
 		if err != nil {
-			return nil, err
+			return nil, util.NewErrBadRequest(fmt.Sprintf("Validation failed: %s", err.Error()))
```

Comment:
I think you can just use `%v` instead of calling `err.Error()`: https://goplay.space/#RO_ix5fIoIJ

Reply:
👍🏻 thanks ... fixed